### DISCUSSION TITLE
Disable status-bar using keybinding

### DISCUSSION THREAD

**Discussion by luiswirth** [zellij-org/zellij#955] [Q&A]
I would like to have a keybinding to disable the status-bar.
Is this possible? E.g. can you disable a plugin using keybindings?

**Comment by imsnif** [comment]
So, erm... yes - this should be possible, but it might be a little tricky to implement (not impossible, but we have some logic overloaded on these, specifically the fullscreen logic). I definitely want to fix those hacks at some point. Until then though: would disabling the status bar entirely be a good solution too, or is it important to you to have it toggle-able at runtime?

**Reply by luiswirth** [reply]
So my specific use case is, to have a keybinding to enter  "zen mode", which hides any distracting UIs. I would like for this to be toggle-able at runtime. For neovim this already work, but not yet for zellij.

**Comment by evnoj** [comment]
I arrived here seeking the same - wondering if this is now possible? I am continuing to investigate, and will post here if I find a solution.

**Comment by martshal** [comment]
I was also wondering about the possibility of a "zen mode".  I frequently use vim in zen mode to remove distractions.  Having a togglable zen mode in zellij would be ideal.

**Comment by maulik13** [comment]
Same here, I find the status bar with shortcuts very helpful and I do prefer it during some operations for dealing with layouts/panes. But, I do not need it all the time. My current workaround is to start the whole session in a different layout each time I want to work in zenish mode. But that gets a bit annoying time to time.

**Comment by haroonalbar** [comment]
+1

**Comment by Zik42** [comment]
Game changer for me, +1

**Comment by jeffs** [comment]
Since nobody explicitly mentioned it yet: Zen mode should also toggle the tab bar, and possibly pane frames.
